docs/src/pages/customization/unstyled-components/unstyled-components.md

```diff
@@ -0,0 +1,138 @@
+# Unstyled components
+
+<p class="description">Implementing a custom design system using Material-UI</p>
+
+Material-UI provides a set of components without any styles. They can be used to implement a custom design system that is not based on Material Design.
+
+The effort to create the unstyled components have started recently, and just a few components have been created yet.
+We intend to focus on the unstyled package extensively in the upcoming months.
+
+## Use cases
+
+Is the unstyled package good for you?
+
+If you:
+
+- need to implement a design system that significantly differs from Material Design
+- need precise control over the rendered HTML of the components
+- need to implement custom components but don't want to deal with accessibility features on your own
+
+Then yes, you will find `material-ui/unstyled` useful.
+
+If, however, you:
+
+- need components that look well out of the box
+- need to customize Material Design with your brand colors
+- need to implement a design system based on Material Design
+
+Then you may be better off using the `@material-ui/core` package and customize it [as described in the docs](../how-to-customize).
+
+## Components vs hooks
+
+The unstyled package has two kinds of building blocks:
+
+- unstyled components
+- hooks
+
+Most of the unstyled components are implemented with the help of a hook (where it makes sense). Hooks encapsulate logic, while components provide structure.
+
+When creating custom components based on the unstyled ones, you can use either unstyled components or hooks.
+
+With components, you will usually be able write less code, but with hooks you have the ultimate control over the structure of the rendered HTML.
```

Comment:
We should somewhere add a note that when using hooks, the a11y is greatly in hands of the developers. Mui is providing the properties that are necessary, but it's their responsibility to create the correct HTML structure and add the necessary properties.

Reply:
Good point, I'll add it.

Reply:
"with great power comes great responsibility" 😀

Reply:
I had this exact quote in mind when writing this 😃